[2024-05-22T01:32:56.077+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-22T01:32:56.145+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_dataset.load manual__2024-05-22T01:16:10.864259+00:00 [queued]>
[2024-05-22T01:32:56.183+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_dataset.load manual__2024-05-22T01:16:10.864259+00:00 [queued]>
[2024-05-22T01:32:56.185+0000] {taskinstance.py:2306} INFO - Starting attempt 6 of 10
[2024-05-22T01:32:56.538+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load> on 2024-05-22 01:16:10.864259+00:00
[2024-05-22T01:32:56.556+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1097) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-22T01:32:56.560+0000] {standard_task_runner.py:63} INFO - Started process 1121 to run task
[2024-05-22T01:32:56.560+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'kaggle_dataset', 'load', 'manual__2024-05-22T01:16:10.864259+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_dataset.py', '--cfg-path', '/tmp/tmpx69zphnp']
[2024-05-22T01:32:56.568+0000] {standard_task_runner.py:91} INFO - Job 22: Subtask load
[2024-05-22T01:32:56.701+0000] {task_command.py:426} INFO - Running <TaskInstance: kaggle_dataset.load manual__2024-05-22T01:16:10.864259+00:00 [running]> on host b17c52120858
[2024-05-22T01:32:56.985+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_dataset' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-05-22T01:16:10.864259+00:00' AIRFLOW_CTX_TRY_NUMBER='6' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-22T01:16:10.864259+00:00'
[2024-05-22T01:32:56.989+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-22T01:32:57.031+0000] {logging_mixin.py:188} INFO - Hola Mundo
[2024-05-22T01:32:57.034+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-22T01:32:57.037+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-22T01:32:57.077+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=kaggle_dataset, task_id=load, run_id=manual__2024-05-22T01:16:10.864259+00:00, execution_date=20240522T011610, start_date=20240522T013256, end_date=20240522T013257
[2024-05-22T01:32:57.140+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-22T01:32:57.198+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-22T01:32:57.205+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
